contracts/modules/SSVClusters.sol

```diff
@@ -357,4 +417,57 @@ contract SSVClusters is ISSVClusters {
             emit ValidatorExited(msg.sender, operatorIds, publicKeys[i]);
         }
     }
+
+    function migrateClusterToETH(uint64[] calldata operatorIds, Cluster memory cluster) external payable override {
+        StorageData storage s = SSVStorage.load();
+        StorageProtocol storage sp = SSVStorageProtocol.load();
+
+        (bytes32 hashedCluster, uint8 version) = cluster.validateHashedCluster(msg.sender, operatorIds, s);
+        ClusterLib.validateClusterVersion(version, CoreLib.VERSION_SSV);
+        cluster.validateClusterIsNotLiquidated();
+
+        uint256 ssvBalance = cluster.balance;
+        // migrate operators to ETH defaults if needed
+        uint64[] memory opIds = operatorIds;
+        for (uint256 i; i < opIds.length; ++i) {
+            ISSVNetworkCore.Operator storage operator = s.operators[opIds[i]];
+            operator.ensureETHDefaults();
+        }
+
+        // compute cluster data using ETH fields
+        (uint64 clusterIndex, uint64 burnRate) = OperatorLib.updateClusterOperators(
+            operatorIds,
+            true,
+            cluster.validatorCount,
```

Comment:
@venimir-ssv We should remove the current `cluster.validatorCount` amount from operators validatorCount (SSV). I see you add the `cluster.validatorCount` to `operator.ethValidatorCount` in `OperatorLib.updateClusterOperators`, but please double check we remove the amount from the operators SSV validatorCount